src/proxy-ui-api/src/test/java/org/niis/xroad/restapi/service/KeyServiceTest.java

```diff
@@ -84,4 +109,26 @@ public void getKey() throws Exception {
         KeyInfo keyInfo = keyService.getKey(GOOD_KEY_ID);
         assertEquals(GOOD_KEY_ID, keyInfo.getId());
     }
+
+    @Test
+    @WithMockUser(authorities = {"VIEW_KEYS", "EDIT_KEYTABLE_FRIENDLY_NAMES"})
+    public void updateKeyFriendlyName() throws Exception {
+        KeyInfo keyInfo = keyService.getKey(GOOD_KEY_ID);
+        assertEquals("friendly-name", keyInfo.getFriendlyName());
+        keyInfo = keyService.updateKeyFriendlyName(GOOD_KEY_ID, "new-friendly-name");
+        assertEquals("new-friendly-name", keyInfo.getFriendlyName());
+    }
+
+    @Test(expected = KeyService.KeyNotFoundException.class)
+    @WithMockUser(authorities = { "EDIT_KEYTABLE_FRIENDLY_NAMES", "VIEW_KEYS" })
+    public void updateKeyFriendlyNameWithoutRights() throws Exception {
```

Comment:
Not sure if I understand the name of this test `updateKeyFriendlyNameWithoutRights`. Seems to have the correct rights `"EDIT_KEYTABLE_FRIENDLY_NAMES", "VIEW_KEYS"`. This seems to be testing the same logic as [updateFriendlyNameUpdateFails](https://github.com/nordic-institute/X-Road/pull/258/files#diff-53e81ff7c330497d688a4d474a8661d7R130)